cloud-movies-be/lib/api-gw/api-gw-stack.ts

```diff
@@ -38,6 +39,18 @@ export class ApiGwStack extends cdk.Stack {
       },
     });
 
+    uploadMovieFn.addToRolePolicy(new iam.PolicyStatement({
+      actions: ['s3:PutObject', 's3:GetObject'],
+      resources: [`${moviesBucket.bucketArn}/*`],
+    }));
```

Comment:
This is excess because you can modify the policy with L3 statements as in line 91. Currently only read permissions are allowed, which must also be expanded to write ones.